src/main/kotlin/be/duncanc/discordmodbot/bot/commands/Spoiler.kt

```diff
@@ -0,0 +1,36 @@
+package be.duncanc.discordmodbot.bot.commands
+
+import be.duncanc.discordmodbot.data.services.UserBlockService
+import net.dv8tion.jda.api.MessageBuilder
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent
+import org.springframework.stereotype.Component
+
+@Component
+class Spoiler(
```

Comment:
What the use of this command you can perfectly use spoiler tags for attachments on mobile?

Reply:
There is? I was not aware, please show me how.

Reply:
I'm not sure if this will work since the original message with the image gets deleted after executing the command also there's a small window were people will be able to see the attachment, so I'm not a fan of resolving it this way.

Discord should just fix this on the mobile client, regardless we will merge it but if seems to not work it will be removed again.

Reply:
Sounds good!